_examples/custom-formatter/go.mod

```diff
@@ -3,6 +3,10 @@ module custom-formatter
 go 1.14
 
 require (
-	github.com/cucumber/godog v0.11.0
+	github.com/cucumber/gherkin-go/v11 v11.0.0 // indirect
+	github.com/cucumber/godog v0.10.1-0.20210705192606-df8c6e49b40b
+	github.com/cucumber/messages-go/v10 v10.0.3 // indirect
 	github.com/cucumber/messages-go/v16 v16.0.1
+	github.com/hashicorp/go-immutable-radix v1.3.1 // indirect
+	github.com/hashicorp/go-memdb v1.3.2 // indirect
```

Comment:
This change pinpoints a not yet released version of `github.com/cucumber/godog` to resolve import dependency on `github.com/cucumber/messages-go/v16`.

Reply:
I don't want this PR to snowball, but is https://github.com/cucumber/godog/issues/389 worth bearing in mind here if we're changing the dependencies?

Reply:
As far as I understood, the idea of #389 is to start using `github.com/cucumber/common/messages/go/v16` packages instead of `github.com/cucumber/messages-go/v16`. 

I think such change should to be delivered in separate PR to change all relevant imports of the `github.com/cucumber/godog`.

Current change is only a fix in one of examples provided by `godog` which is organized as a separate module and so does not inherit parent (`godog`) module dependencies. Previously this example was using `godog` `v0.11.0`, but at the same was using `messages-go/v16` that was only supported by newer (untagged) version of `godog`. 

In this situation we have 3 options in my opinion:
* revert example to use `messages-go/v10` so that  it is again compatible with `godog` `v0.11.0`,
* update example to use pinned commit for newer `godog` (as done in this PR),
* remove `go.mod` from example, so that it inherits parent dependencies.

Reply:
> As far as I understood, the idea of #389 is to start using github.com/cucumber/common/messages/go/v16 packages instead of github.com/cucumber/messages-go/v16.

That's right

I think such change should to be delivered in separate PR to change all relevant imports of the github.com/cucumber/godog.

👍